packages/paste-core/components/display-heading/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "@twilio-paste/display-heading",
+  "version": "0.0.0",
+  "category": "typography",
+  "status": "production",
+  "description": "Display heading is large, expressive text that stands out from traditional headings and is used for marketing and attention grabbing purposes.",
+  "author": "Twilio Inc.",
+  "license": "MIT",
+  "main:dev": "src/index.tsx",
+  "main": "dist/index.js",
+  "module": "dist/index.es.js",
+  "types": "dist/index.d.ts",
+  "sideEffects": false,
+  "publishConfig": {
+    "access": "public"
+  },
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "yarn clean && NODE_ENV=production node build.js && tsc",
+    "build:js": "NODE_ENV=development node build.js",
+    "build:props": "typedoc --tsconfig ./tsconfig.json --json ./dist/prop-types.json",
+    "clean": "rm -rf ./dist",
+    "tsc": "tsc"
+  },
+  "peerDependencies": {
```

Comment:
Suggestion(blocking): need to add text as a peer/dev dep (and all of text's deps)